app/scripts/views/museum/tab-households-partial.html

```diff
@@ -7,35 +7,35 @@
     </div> <!-- /.col -->
     <div class="col-xs-4">
         <div class="chart-graph">
-            <h4>Household Types</h4>
-            <div id="pie-chart-household-type"><svg></svg></div>
+            <h4>Median Household Income</h4>
+            <span class="numerical">{{ house.data['B19013_001E'].popAvg | currency:"$":0 }}</span>
```

Comment:
It's an ACS var with the id `B19013_001E`. Nothing here has been changed as github suggests. It's just been reordered for layout purposes.